src/devices/moes.ts

```diff
@@ -388,6 +388,24 @@ const definitions: Definition[] = [
             device.save();
         },
     },
+    {
+        fingerprint: [{modelID: 'TS0012', manufacturerName: '_TZ3000_18ejxno0'}],
```

Comment:
Please add this as a new `tuya.whitelabel` to `TS0012` instead.

Reply:
Sorry but I am new to this and don't really understand what to do...